src/main/java/hudson/remoting/Channel.java

```diff
@@ -832,6 +854,11 @@ public void setJarCache(@Nonnull JarCache jarCache) {
      */
     public <V,T extends Throwable>
     V call(Callable<V,T> callable) throws IOException, T, InterruptedException {
+        if (isClosingOrClosed()) {
+            // No reason to even try performing a user request
+            throw new IOException("Remote call on " + name + " failed. The channel is closing down or has closed down");
```

Comment:
I think from the user (~ plugin author) perspective either the channel is open for business, or it is closed and the request cannot be satisfied. There is no reason to distinguish a channel in the process of closing.